routes/flock.json

```diff
@@ -119,43 +119,282 @@
+      },
+      {
+        "title": "Battery and Battery Management",
+        "content": [
+          "The battery management system (BMS) is used to monitor and balance the cell array used to power the Linear Induction Motors and low voltage systems. The lithium ion cells experience multiple charge and discharge cycles, thus they will degrade if improperly managed without the BMS. The BMS also utilizes passive cell balancing to ensure that all batteries in the stack are evenly charged, thereby prolonging cell life and maintaining healthy charge levels."
+        ],
+        "parallax": "/images/flock/goose5/battery.png"
+      },
+      {
+        "title": "Embedded Sensors",
+        "content": [
+          "The pod consists of an array of 32 sensors, with 11 unique models. A few examples of models would be the Piezo, which measures vibrations both at the wheel and frame level, and the Phototransitor that reads the RPM of the pod's wheels for calculating speed."
+        ]
+      },
+      {
+        "title": "Physical System Architecture: Communications",
+        "content": [
+          "There are 2 major components comprising the communications system:",
+          "<ul><li>The Networking and TCP component utilizes a Desktop Raspberry Pi. It facilitates relaying sensor data from the sensors to the desktop dashboard to be displayed, and to send commands from the remote desktop dashboard back to the pod.</li>",
+          "<li>The CAN Bus connects a Raspberry Pi with Arduinos/Sensors. It serves as a cost-effective way of transmitting data between the devices with minimal interference from other subsystems in the pod.</li></ul>"
+        ]
+      },
+      {
+        "title": "User Interface: Dashboard",
+        "content": [
+          "Developed using QT Designer, QML and C++, the desktop system's dashboard displays sensor data about the pod's performance and provides a platform to control the pod remotely. The dashboard is organized in containers, where each container displays information regarding similar sensor data in an easily interpreted fashion. Gauges show the danger levels associated with a reading, and are thus used extensively. The colour warnings and indicators are designed to easily stand out to the user, thereby warning them to take action more effectively.",
+          "As the final link in starting the execution of the pod run, the pod starts when the user presses the \"start\" button, after which all subsequent operations are autonomous. However, the user has the option to enter \"Manual Mode\", where the user can then activate brakes and control the LIM operating frequency."
+        ],
+        "parallax": "/images/flock/goose5/dashboard.png"
+      }
+    ]
+  },
+  {
+    "name": "Goose X",
+    "thumbnail": "/images/flock/goosex.png",
+    "url": "goose_x",
     "desc": "",
-    "banner": "/images/flock/goose3/torontorender.jpg",
+    "banner": "/images/flock/goosex/torontorender.jpg",
```

Comment:
Why are you changing goose3 to gooseX?

Reply:
The image is supposed to be used for GooseX to begin with.